.github/workflows/check-alerts.yml

```diff
@@ -6,12 +6,18 @@ on:
   # Have the ability to trigger this job manually through the API
   workflow_dispatch:
 
+
 jobs:
   update-disabled-tests:
     runs-on: ubuntu-18.04
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Install requests
+        run: |
+           pip install requests
       - name: Check for alerts and creates issue
         run: |
-          .github/scripts/check_alerts.py
+          python3 torchci/scripts/check_alerts.py
```

Comment:
Nit: I know that python and python3 are kind of the same for us now, but there is a small inconsistency here in pip/python and pip3/python3 pairs.  In systems where older python 2 is still around, pip basically installs deps for the python 2

Reply:
Yea I was surprised that it didn't work with just using normal python. I had to use python3 on this :|